src/components/Login/index.tsx

```diff
@@ -1,16 +1,42 @@
 import React, { FormEvent, useState } from 'react';
+
 import styles from './Login.module.css';
 
 export default function Login(): React.ReactElement {
-    const [email, setEmail] = useState<string | undefined>();
-    const [password, setPassword] = useState<string | undefined>();
+    const [details, setDetails] = useState<userDetails>({
+        email: '',
+        password: '',
+    });
+    type userDetails = {
+        email: string;
+        password: string;
+    };
+
+    function login(details: userDetails): void {
+        console.log(details);
 
-    function validateForm() {
-        return email && password;
+        if (details.email === 'user@user.com' && details.password === 'user') {
+            console.log('logged in');
+            localStorage.setItem('isLogged', 'true');
+        } else if (
+            details.email === 'admin@admin.com' &&
+            details.password === 'admin'
+        ) {
+            // localStorage.setItem('isLogged', 'false')
+
+            console.log('logged in');
+            localStorage.setItem('isLogged', 'true');
+            localStorage.setItem('admin', 'true');
+        } else {
+            localStorage.setItem('isLogged', 'false');
+            localStorage.setItem('admin', 'false');
+        }
     }
 
     function handleSubmit(event: FormEvent<HTMLFormElement>) {
         event.preventDefault();
+
+        login(details);
```

Comment:
after logging in we should redirect the user to the homepage
You can achieve it by adding this at the top of this component
```
const navigate = useNavigate();
```
and then calling `navigate('/')` after login